unshare.c

```diff
@@ -56,6 +56,7 @@ static PyMethodDef methods[] = {
         "CLONE_NEWPID\n"
         "  CLONE_NEWNET\n"
     },
+#if __GLIBC_MINOR__ >= 14
```

Comment:
This will break with [glibc 3.0](https://gcc.gnu.org/wiki/cauldron2017#glibc30). You should check `__GLIBC__` (the major version), too.